var/spack/repos/builtin/packages/py-bluepyefe/package.py

```diff
@@ -10,21 +9,17 @@ class PyBluepyefe(PythonPackage):
     """Blue Brain Python E-feature extraction"""
 
     homepage = "https://github.com/BlueBrain/BluePyEfe"
-    pypi = "bluepyefe/bluepyefe-0.3.13.tar.gz"
+    pypi = "bluepyefe/bluepyefe-2.2.18.tar.gz"
     git = "https://github.com/BlueBrain/BluePyEfe.git"
 
     version("2.2.18", sha256="bfb50c6482433ec2ffb4b65b072d2778bd89ae50d92dd6830969222aabb30275")
 
     depends_on("py-setuptools", type="build")
 
-    depends_on("py-numpy", type=("build", "run"))
+    depends_on("py-numpy@:1.23", type=("build", "run"))
     depends_on("py-neo", type=("build", "run"))
     depends_on("py-matplotlib", type=("build", "run"))
     depends_on("py-efel", type=("build", "run"))
     depends_on("py-scipy", type=("build", "run"))
     depends_on("py-h5py", type=("build", "run"))
     depends_on("py-igor", type=("build", "run"))
-
-    def setup_run_environment(self, env):
-        env.set("NEURON_INIT_MPI", "0")
-        env.unset("PMI_RANK")
```

Comment:
I'd like to note that either (a) this is not needed any longer or (b) it may come back to bite us.

We may have to set those variables at module generation time via the module config.

Reply:
https://github.com/spack/spack/pull/40592#discussion_r1365302697

It was mentioned here as being safe to remove, so we should be good.